xsl/gnc-custom-chm.xsl

```diff
@@ -0,0 +1,66 @@
+<?xml version='1.0'?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                version='1.0'>
+
+
+<!-- import stylesheet 
+    <xsl:import href="https://cdn.docbook.org/release/xsl/current/htmlhelp/htmlhelp.xsl" />
+    http://docbook.sourceforge.net/release/xsl/ is deprecated, but we have to 
+    use it for compatibility.
+-->
+<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/htmlhelp/htmlhelp.xsl" />
+
+
+<!-- parameters for GnuCash docs. -->
+<xsl:param name="admon.graphics.extension">.png</xsl:param>
+<xsl:param name="callout.graphics.extension">.png</xsl:param>
+
+<xsl:param name="htmlhelp.default.topic">ch01.html</xsl:param>
+
+<xsl:param name="generate.toc">
+book      nop
+</xsl:param>
+
+<!-- language specific character code for wchar
+     ja: Shift_JIS
+     ru: KOI8-R
+-->
+
+<xsl:param name="htmlhelp.encoding">
+  <xsl:choose>
+    <xsl:when test="$gnc.lang = 'ja'">Shift_JIS</xsl:when>
+    <xsl:when test="$gnc.lang = 'ru'">KOI8-R</xsl:when>
+    <xsl:otherwise>ISO-8859-1</xsl:otherwise>
```

Comment:
ISO-8859-1 is missing € and it is often used.
Can htmlhelp not use UTF8?

Reply:
Yes and no.

The UTF-8 may work in the HTML pane because "iexplore.exe" renders it. But HTML HELP renders the TOC pane.
THE HTML HELP uses legacy MS codepages, such as 932(ja), 855(ru), 1252(latin). So the EURO sign may not be rendered properly.

The current default setting for CHM is ISO-8859-1 and I can't expect what happens if I set it UTF-8.

Reply:
> THE HTML HELP uses legacy MS codepages, such as 932(ja), 855(ru), 1252(latin). So the EURO sign may not be rendered properly.

It's about 12 years past time to get rid of the chm build and open the documentation in the default browser or in a gtkwebkitwebview. That's obviously out of scope for this PR and not something that I'd ask of you.